recipe/meta.yaml

```diff
@@ -39,22 +39,20 @@ requirements:
     - expat 2.4.9
     - fontconfig 2.14
     - freetype 2.10
-    - icu 68  # [s390x or aarch64 or (osx and arm64)]
-    - icu 58  # [not (s390x or aarch64 or (osx and arm64))]
+    - icu {{ icu }}
     - jpeg 9e
     - libiconv 1.16     # [osx]
     - libpng 1.6.*
     - libtiff 4.1.*  # [not ((osx and arm64) or (linux and aarch64))]
     - libtiff 4.2.*  # [(osx and arm64) or (linux and aarch64)]
     - libwebp 1.2
-    - libwebp-base
+    - libwebp-base 1.2
     - zlib 1.2.*
   run:
     - expat >=2.4.9,<3.0a0
     - fontconfig >=2.14.1,<3.0a0
     - freetype >=2.10.4,<3.0a0
-    - icu >=68.1,<69.0a0  # [s390x or aarch64 or (osx and arm64)]
-    - icu >=58.0,<59.0a0  # [not (s390x or aarch64 or (osx and arm64))]
+    - icu >=73.1,<74.0a0
```

Comment:
would icu `{{ pin_compatible('icu') }}` possibly avoid the hassle of doing this manually?

Reply:
Explicit is better than implicit for the runtime dependencies (and in general anyway).